book.bib

```diff
@@ -2418,7 +2418,9 @@ @website{import.io
 @misc{analyze_viz,
 author ={Tableau Software},
 title ={Visual Analysis Best Practices, Simple Techniques for Making Every Data Visualization Useful and Beautiful},
-url={http://www.dataplusscience.com/files/visual-analysis-guidebook.pdf}
+howpublished = "\url{http://www.dataplusscience.com/files/visual-analysis-guidebook.pdf}"'
```

Comment:
I think you are missing a comma at the end of this line. 
@strivedi2 what is the difference between url and howpublished?

Reply:
There is a different format for adding reference for websites. Here is one of the links with the format: https://coderwall.com/p/wntyia/how-to-cite-a-website-in-latex